src/ralph/lib/transitions/tests/__init__.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from django.contrib.contenttypes.models import ContentType
+from django.test import TestCase
+
+from ralph.lib.transitions.forms import TransitionForm
+from ralph.lib.transitions.models import Transition, Action
+
+from ralph.tests.models import Order
+
+
+class TransitionTestCase(TestCase):
+    def is_identical_list(self, l1, l2):
```

Comment:
you could use https://docs.python.org/3.4/library/unittest.html#unittest.TestCase.assertCountEqual instead